src/drc/api/serializers.py

```diff
@@ -54,6 +59,18 @@ def __init__(self, view_name: str = None, *args, **kwargs):
     def get_file_extension(self, filename, decoded_file):
         return "bin"
 
+    def to_internal_value(self, base64_data):
```

Comment:
@sergei-maertens ik weet niet hoe ik dit goed moet oplossen, aangezien er in `Base64FileField` niet gekeken wordt naar wat voor error er terugkomt bij het decoden van de data (geeft altijd invalid file o.i.d. terug)

Reply:
it's not an elegant solution, but it's functional and this is a reference implementation... so it's good :+1: